Release Highlights 2.5 (#2157)
FlowFuse/website: src/blog/2024/06/flowfuse-2-5-release.md

---
title: "FlowFuse 2.5: New features to visualize snapshots, LDAP integration, and more"
subtitle: Enhancing security, visualization, and deployment flexibility.
description: Discover the new features in FlowFuse 2.5, including LDAP integration, visual snapshot comparisons, blueprint previews, snapshot import/export support, and custom domain deployment for dashboards and APIs.
date: 2024-06-07
authors: ["grey-dziuba"]
image: /blog/2024/06/images/release-2-5-graphic.png
tags:
   - posts
   - flowfuse

>    - releases

Reviewer: ```suggestion
   - releases
   - LDAP
   - snapshot
   - blueprints
   - Node-RED
   
```

---

> FlowFuse 2.5 introduces LDAP integration, Snapshot comparison, Extends the ability to preview flow to Blueprints, rounds out the management for snapshots, while lastly enabling customer to deploy FlowFuse with your own domain name.

Reviewer 2: ```suggestion
FlowFuse 2.5 introduces LDAP integration, snapshot comparison, extends the ability to preview flow to Blueprints, rounds out the management for snapshots, and allows you to point your own domain names at your FlowFuse instances.
```

<!--more-->

## Support for LDAP integration

> In our commitment to making FlowFuse more versatile and secure, we've introduced support for LDAP. This has been a much requested feature that we are excited to release.  This allows organizations to manage user authentication and authorization seamlessly, leveraging their existing LDAP infrastructure. With this feature, you can ensure that access controls are robust and aligned with your company's security policies.

Reviewer 2: ```suggestion
In our commitment to making FlowFuse more versatile and secure, we've introduced support for LDAP alongside our existing SAML SSO options. This has been a much requested feature that we are excited to release.  This allows organizations to manage user authentication seamlessly, leveraging their existing LDAP infrastructure. With this feature, you can ensure that access controls are robust and aligned with your company's security policies.
```

Technically, we only support authentication.

## Compare Snapshots Visually to see differences

Keeping track of changes and updates has never been easier. With our new visual comparison tool, you can now compare snapshots and see the differences at a glance. This feature provides a clear, graphical representation of changes, making it simple to identify modifications and understand their impact.

## Preview Blueprints before Deployment

> Similarly to the feature to be able to compare snapshots we have enabled this same feature to allow users to preview team flows and blueprints prior to deploying.  This expedites the process when you are attempting to find the correct flow for your application.  We find that our users will be able to 

Reviewer: missing content at the end

## Import and Export support for snapshots

> Managing your Node-RED artifacts just got simpler with our new import and export support for snapshots. You can now easily transfer configurations between deployment or back up your current setup with just a few clicks. This feature is perfect for those who need to replicate environments within or ensure their configurations are safely stored.

Reviewer: within what?

   - releases
   - LDAP
   - snapshot
   - blueprints
   - Node-RED
   

---

FlowFuse 2.5 introduces LDAP integration, snapshot comparison, extends the ability to preview flow to Blueprints, rounds out the management for snapshots, and allows you to point your own domain names at your FlowFuse instances.

<!--more-->

## Support for LDAP integration

In our commitment to making FlowFuse more versatile and secure, we've introduced support for LDAP alongside our existing SAML SSO options. This has been a much requested feature that we are excited to release.  This allows organizations to manage user authentication seamlessly, leveraging their existing LDAP infrastructure. With this feature, you can ensure that access controls are robust and aligned with your company's security policies.

## Compare Snapshots Visually to see differences

Keeping track of changes and updates has never been easier. With our new visual comparison tool, you can now compare snapshots and see the differences at a glance. This feature provides a clear, graphical representation of changes, making it simple to identify modifications and understand their impact.

## Preview Blueprints before Deployment

> Similarly to the feature to be able to compare snapshots we have enabled this same feature to allow users to preview team flows and blueprints prior to deploying.  This expedites the process when you are attempting to find the correct flow for your application.  We find that our users will be able to 

Author: ```suggestion
Similarly to the feature to be able to compare snapshots we have enabled this same feature to allow users to preview team flows and blueprints prior to deploying.  This expedites the process when you are attempting to find the correct flow for your application.  We find that our users will be able to expedite the exploration of both their own team library and the blueprints.
```

## Import and Export support for snapshots

> Managing your Node-RED artifacts just got simpler with our new import and export support for snapshots. You can now easily transfer configurations between deployment or back up your current setup with just a few clicks. This feature is perfect for those who need to replicate environments within or ensure their configurations are safely stored.

Author: ```suggestion
Managing your Node-RED artifacts just got simpler with our new import and export support for snapshots. With just a few clicks, you can easily transfer configurations between deployments or back up your current setup. This feature is perfect for those who need to replicate environments within their own enterprise with multiple deployments or ensure their configurations are safely stored.
```

Author: Feels redundant now.
